res/css/views/rooms/wysiwyg_composer/components/_LinkModal.pcss

```diff
@@ -16,14 +16,31 @@ limitations under the License.
 
 .mx_LinkModal {
     padding: $spacing-32;
-
-    .mx_Dialog_content {
-        margin-top: 30px;
-        margin-bottom: 42px;
-    }
+    max-width: calc(600px - ($spacing-32 * 2));
+    height: calc(341px - ($spacing-32 * 2));
```

Comment:
I think `box-sizing: border-box;` would make this simpler. Then you could just do `max-width: 600px; height; 341px;`, in theory

Reply:
Oh that's perfect and it's working. Simpler and cleaner, thanks !